README.md

```diff
@@ -15,39 +15,72 @@
   <img src="assets/res.webp"/>
 </p>
 
-# About
-
 This GTK theme is based on the [Colloid](https://github.com/vinceliuice/Colloid-gtk-theme) theme made by [Vinceliuice](https://github.com/vinceliuice)
 
-## Usage
+# Usage
 
 ### Requirements
 
-- GTK `>=3.20`
+- GTK >=3.20
+- `python3`
 - `gnome-themes-extra` (or `gnome-themes-standard`)
 
-### Installation
+If you want to build the theme, make sure these packages are installed:
+- `sassc`
+- `inkscape`
+- `optipng`
+
+In order to make sure you're running Python 3, run ``python --version`` and it should output ``Python 3.x.x``
```

Comment:
Don't think this needs explained, we'll ask users to run with `python3` to ensure they're on python 3. We don't care about the minor or patch versions for this usecase.
```suggestion
```

Reply:
alr